### ISSUE TITLE
Scirius can't get data from ES 6.2.3

### ISSUE THREAD

**Issue by v0yager** [StamusNetworks/scirius#142]
Hello,guys.
I am new to this, so I ask for an apology if the question is too easy.

Scirius can't get data from ES 6.2.3 !
But the status of the ES is green light, I can see the indexes in the scirius UI.

I think that scirius query in ES failed ,maybe~
what name I should set  in the (scirius -> suricata -> edit), I am confused。

Anyone can help me ,thanks a lot!

**Comment by pevma** [comment]
Try adjusting the following : 
```
ELASTICSEARCH_VERSION = 6
ELASTICSEARCH_KEYWORD = "keyword"
KIBANA_VERSION = 6
KIBANA_INDEX = ".kibana"
KIBANA_URL = "http://localhost:5601"
```

in `/etc/scirius/local_settings`

**Comment by v0yager** [comment]
I tried ，but failed。
Scirius can jump the  page of Kibana dashboard。
But there is no ES data in Scirius UI
ElasticSearch like this：
USE_ELASTICSEARCH = True
ELASTICSEARCH_ADDRESS = "10.3.220.113:9200"
ELASTICSEARCH_LOGSTASH_INDEX = "logstash-*"
ELASTICSEARCH_LOGSTASH_ALERT_INDEX = ELASTICSEARCH_LOGSTASH_INDEX
ELASTICSEARCH_LOGSTASH_TIMESTAMPING = "daily"
ELASTICSEARCH_VERSION = 6
ELASTICSEARCH_KEYWORD = "keyword"
ELASTICSEARCH_HOSTNAME = "COMDEV0585"

 Kibana like this：
USE_KIBANA = True
KIBANA_PROXY = True
KIBANA_URL = "http://10.3.220.113:5601"
KIBANA_INDEX = ".kibana"
KIBANA_VERSION= 6
KIBANA_DASHBOARDS_COUNT = 20
KIBANA_DASHBOARDS_PATH = '/opt/kibana-dashboards/'

**Comment by regit** [comment]
Can yoy try to set the remote elasticsearch address in the interface (in Stamus Icon-> settings).

**Comment by v0yager** [comment]
I have set the remote ES address .But ,there is no data in Scirius UI
![image](https://user-images.githubusercontent.com/8177238/43626058-d1748400-9722-11e8-8a70-aafa3c209016.png)
The eve.json log like this in ES
![image](https://user-images.githubusercontent.com/8177238/43627151-bf99855a-9727-11e8-9d37-5b5106015bcb.png)

**Comment by pevma** [comment]
unselect "use elasticsearch" - so you only leave the "custom" option? Any luck? (plus clear browser cache just in case)

**Comment by v0yager** [comment]
I tried  whether  selecting "use custom ES" option, Scirius can get index and cluster info from ES,and the status of ES is yellow or green. But there is no ES data in Scirius UI
So maybe other problems~

I also set the time-zone as  localtime and change the name of the Suricata (scirius -> suricata -> edit) as the hostname in ES events.
Maybe the problem is the hostname? Thanks a log agian~~
![image](https://user-images.githubusercontent.com/8177238/43651213-5a3a55e8-9774-11e8-8748-de727d5c27bf.png)

**Comment by nyb123456nyb** [comment]
@v0yager  hi:   
   I  faced the problem samed with you, haved you solved it ?  thanks

**Comment by pevma** [comment]
Think you would need to adjust the address of the elasticsearch in local_settings as well, did you do that ?

**Comment by nyb123456nyb** [comment]
Yes ,i do. now all status is green ,but no data。
![image](https://user-images.githubusercontent.com/21351974/62201445-045cf280-b3ba-11e9-8182-5b9f79b1e8b1.png)
![image](https://user-images.githubusercontent.com/21351974/62201455-0c1c9700-b3ba-11e9-8a02-4faa42ae45a9.png)

**Comment by pevma** [comment]
Whats is your `local_settings.py ` look like ?

**Comment by ipworkx** [comment]
Same problem here. What is the solution?

**Comment by pevma** [comment]
Whats is your local_settings.py look like ? Inside you should probably set the hostname/es variables correct if not already done. Something similar to  - https://github.com/StamusNetworks/SELKS/issues/201#issuecomment-538728806   depending if ES is local or remote.

**Comment by pevma** [comment]
Ok - let us know hoe it goes!

**Comment by ipworkx** [comment]
My local_settings.py is like:

USE_ELASTICSEARCH = True
ELASTICSEARCH_KEYWORD = "keyword"
ELASTICSEARCH_LOGSTASH_INDEX = "suricata-*"
ELASTICSEARCH_LOGSTASH_ALERT_INDEX = "suricata-alert-"
ELASTICSEARCH_LOGSTASH_TIMESTAMPING = "daily"
ELASTICSEARCH_VERSION = 6
ELASTICSEARCH_KEYWORD = "keyword"
ELASTICSEARCH_HOSTNAME = "u002.system.ipworkx.loc"
ELASTICSEARCH_ADDRESS = "u002.system.ipworkx.loc:9200"


USE_KIBANA = True
KIBANA_PROXY = True
KIBANA_DASHBOARDS_PATH = '/opt/kibana-dashboards/'
KIBANA_VERSION = 6
KIBANA_INDEX = ".kibana_1"
KIBANA_URL = "http://u002.system.ipworkx.loc:5601"
KIBANA6_DASHBOARDS_PATH = "/opt/selks/kibana6-dashboards/"

#SURICATA_UNIX_SOCKET = "/var/run/suricata/suricata-command.socket"
SURICATA_NAME_IS_HOSTNAME = False

USE_EVEBOX = True
EVEBOX_ADDRESS = "u002.system.ipworkx.loc:5636"

USE_SURICATA_STATS = True
USE_LOGSTASH_STATS = True

ALLOWED_HOSTS=["*"]
STATIC_ROOT="/var/lib/scirius/static/"

DATABASES = {
  'default': {
     'ENGINE': 'django.db.backends.sqlite3',
     'NAME': os.path.join(BASE_DIR, 'db', 'db.sqlite3'),
  }
}
DBBACKUP_STORAGE_OPTIONS = {'location': '/var/backups/'}


My logstash output config is like:

output {
    if [event_type] == "stats"  {
      elasticsearch {
        hosts => [ "172.16.32.2:9200", "172.16.32.5:9200", "172.16.32.7:9200" ]
        index => "suricata-stats-%{+YYYY.MM.dd}"
        template => "/etc/logstash/conf.d/suricata/template/suricata-stats-template.json"
        template_name => "suricata-stats"
        template_overwrite => true
      }
    } else {
      if [event_type] {
        elasticsearch {
          hosts => [ "172.16.32.2:9200", "172.16.32.5:9200", "172.16.32.7:9200" ]
          index => "suricata-%{event_type}-%{+YYYY.MM.dd}"
          template => "/etc/logstash/conf.d/suricata/template/suricata-template.json"
          template_name => "suricata"
          template_overwrite => true
        }
      }
    }

**Comment by ipworkx** [comment]
I'm running a SELKS box using filebeat and outputting to kafka, From kafka to Elastic. Kibana works fine

**Comment by pevma** [comment]
The ES address is reachable form Scirius ?

**Comment by ipworkx** [comment]
Yep my Scirius machine is running ip 172.16.32.9 which is in the same network.
Scirius is seeing all indices from the cluster. All is green and looks fine. I guess.

What values has to be entered in the suricata screen in scirius?
Currently I have:
Name: u002.system.ipworkx.loc
Description: u002.system.ipworkx.loc
Rules Directory: /etc/suricata/rules
Suricata Configuration File: /etc/suricata/suricata.yaml
.
.
.
Is there a way I can t-shoot anywhere?

**Comment by ipworkx** [comment]
The hunt screen with alerts seems to show alerts. So that's fine, although the dashboard itself does not show anything. Item signatures is also fine.
Hope this might give you a clue?

**Comment by ipworkx** [comment]
Could it be that scirius is looking for a host field in the alert indice?
The original host field is now called fields.host due to filebeat and logstash doesn't want to process the host field because of its special purpose

**Comment by pevma** [comment]
can you try with 
`ELASTICSEARCH_LOGSTASH_ALERT_INDEX = "suricata-alert-*"`   
then restart Scisiurs/Kibana
